components/definitions/rewards.d.ts

```diff
@@ -99,6 +99,7 @@ declare namespace Rewards {
     rate: number
     autoContributeChoice: number
     autoContributeChoices: number[]
+    payoutStatus: any
```

Comment:
Similar to the new tab types, we don't want `any` but this could be `undefined` after upgrade. In the Rewards page case, there's a "version" property that can be bumped to clear the `localStorage` cache on upgrade.

Reply:
Thanks, I bumped the version number.